docs/tools/cli/service.rst

```diff
@@ -287,12 +287,74 @@ Service M3 commands
 ``avn service maintenance-start``
 '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-Start service maintenance updates
+Starts the service maintenance updates. 
+
+.. Warning::
+
+  Maintenance updates do not typically cause any noticeable impact on the service in use but may sometimes cause a short period of lower performance or downtime which shall not exceed 1 hour.
+
+
+.. Note::
+  
```

Comment:
Stacking the admonitions looks a bit strange in the output. Perhaps we could move the note to after the command example?

Reply:
Fixed!